torch/_inductor/kernel/flex_attention.py

```diff
@@ -561,9 +584,21 @@ def flex_attention(*args, **kwargs):
         sparse_kv_num_blocks,
         sparse_kv_indices,
     ] + list(other_buffers)
+    num_blocks_for_autotuning = min(10, sparse_kv_indices.get_size()[-1])
```

Comment:
can you explain this line?

Reply:
oh yeah, I was gonna add a comment.

Reply:
https://github.com/pytorch/pytorch/pull/129915/files https://github.com/pytorch/pytorch/pull/129915/files#diff-7a9af76999e838f26a7171771bae1dd3243aefeed3827a8412f2f19fbf7b8caaR459